Formula/feedgnuplot.rb

```diff
@@ -3,8 +3,7 @@ class Feedgnuplot < Formula
   homepage "https://github.com/dkogan/feedgnuplot"
   url "https://github.com/dkogan/feedgnuplot/archive/v1.55.tar.gz"
   sha256 "1205afedf8ce79d8531e0d0f8f9565df365a568a0ee6a8e17738602682095303"
-  # licensed under either "GPL-3.0" or "Artistic-1.0"
-  license "GPL-3.0"
+  license ["GPL-3.0", "Artistic-1.0"]
```

Comment:
I think this should be `Artistic-1.0-Perl`. The [`LICENSE`](https://github.com/dkogan/feedgnuplot/blob/c8dac15db3b409c9a69b11b017dae2d838d3e4da/LICENSE#L8) file directs to http://dev.perl.org/licenses/artistic.html which is the "perl version"